packages/acceptance-tests/pkg-tests-specs/sources/commands/version.test.js

```diff
@@ -196,5 +198,71 @@ describe(`Commands`, () => {
         });
       }),
     );
+
+    test(
+      `it should correctly report a dependent workspace when unable to upgrade its version.`,
+
+      makeTemporaryEnv(
+        {private: true,
+          workspaces: [
+            `packages/*`,
+          ],
+        },
+
+        async ({path, run, source}) => {
+        // Create the primary package.
+          const pkgPrimary = ppath.join(path, `packages/pkg-primary`);
+          await xfs.mkdirpPromise(pkgPrimary);
+          await xfs.writeJsonPromise(ppath.join(pkgPrimary, `package.json`), {
+            name: `pkg-primary`,
+            version: `1.0.0`,
+          });
+
+          // Create the dependant package.
+          const pkgDependant = ppath.join(path, `packages/pkg-dependant`);
+          await xfs.mkdirpPromise(pkgDependant);
+          await xfs.writeJsonPromise(ppath.join(pkgDependant, `package.json`), {
+            name: `pkg-dependant`,
+            version: `1.0.0`,
+            dependencies: {
+              [`pkg-primary`]: `workspace:*`,
+            },
+          });
+
+          // Ensure we have the appropiate plugin.
+          await writeFile(`${path}/.yarnrc.yml`, [
+            `plugins:`,
+            `  - ${JSON.stringify(require.resolve(`@yarnpkg/monorepo/scripts/plugin-version.js`))}`,
+
+          ].join(`\n`));
+
+          let stdout;
+          let stderr;
+          let code;
+
+          try {
+            // Ensure everything is in place.
+            await run(`install`);
+
+            // Execute a version patch.
+            ({code, stdout, stderr} = await run(`workspace`,`pkg-primary`,`version`, `patch`));
```

Comment:
Think we need to update the eslint config a bit cc @paul-soporan

Reply:
#2293

Reply:
I enabled some more rules in https://github.com/yarnpkg/berry/pull/2293 as well